src/frontend/scss/objects/_course_glimpses.scss

```diff
@@ -196,47 +196,9 @@ $course-glimpse-content-padding-sides: 0.7rem !default;
     r-theme-val(course-glimpse, footer),
     $apply-border: true
   );
-
-  &__date,
-  &__cta {
-    display: flex;
-    @include sv-flex-cell-width(50%);
-    flex-direction: column;
-    align-items: flex-start;
-    justify-content: center;
-    margin: 0;
-    padding: 0.5rem 0;
-    align-content: center;
-    font-weight: bold;
-    line-height: 1.1;
-  }
-
-  &__cta {
-    display: flex;
-    align-items: flex-end;
-
-    & > .button {
-      padding: 0;
-      color: white;
-      font-weight: 700;
-
-      &:hover {
-        color: white;
-      }
-
-      & > .icon {
-        display: inline-block;
-        height: 1rem;
-        width: 1rem;
-        margin-left: 0.2rem;
-      }
-    }
-  }
-
-  &__date {
-    // Force a line-break between the meaningful text and the actual date information
-    & > span {
-      display: table;
-    }
-  }
+  margin: 0;
```

Comment:
No please, leave this stylesheet rules, i know you tried to remove the call_to_action, so at least if it is removed, you can drop the `**__cta` rules, but i want to keep everything related to `**__date`